mixin/lib/civimix-schema@5/src/SchemaHelperInterface.php

```diff
@@ -27,6 +27,14 @@
  * @method bool createEntityTable(string $filePath)
  * @method bool alterSchemaField(string $entityName, string $fieldName, array $fieldSpec)
  *
+ * [[ CiviCRM 6.10+ / civimix-schema@6.10+ ]]
```

Comment:
@totten so based on the Civix PR is this supposed to say "civimix-schema@5.93+"? And where did that number come from?

Reply:
@colemanw Yep.